fluentui_icons/build.gradle

```diff
@@ -47,11 +46,13 @@ android {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation "androidx.compose.ui:ui:$composeVersion"
+    //Compose BOM
+    implementation platform("androidx.compose:compose-bom:$composeBomVersion")
```

Comment:
Do we need this here?

Reply:
Yes, needed for the - androidx.compose.ui:ui dependency

Reply:
This wasnt used in other modules

Reply:
Other modules take a dependency on fluentui_core and get bom from there. fluentui_icon does not have this dependency (does not require) so bom had to be added separately.